keps/sig-cloud-provider/2436-controller-manager-leader-migration/README.md

```diff
@@ -349,18 +354,131 @@ feature without providing a configuration, the default configuration will reflec
+    - Impact of its degraded performance or high-error rates on the feature: delayed or retried operations of leader election.
+
+### Scalability
+
+###### Will enabling / using this feature result in any new API calls?
+
+Leader Migration uses exactly one more resource of `coordination.k8s.io/v1.Lease` using the standard leader election process.
+Both `kube-controller-manager` and `cloud-controller-manager` will create, update, and watch on the lease.
+
+If the service accounts are not granted access to the lease resources, the RBAC roles of each controller manager may need to modified before the upgrade.
+
+###### Will enabling / using this feature result in introducing new API types?
+
+Type: `controllermanager.config.k8s.io/v1alpha1.LeaderMigrationConfiguration`
+This resource is only for configuration file parsing. The resource should never reach the API server.
+
+###### Will enabling / using this feature result in any new calls to the cloud provider?
+
+No.
+
+###### Will enabling / using this feature result in increasing size or count of the existing API objects?
+
+This feature uses exactly one more `coordination.k8s.io/v1.Lease` resource. The RBAC roles of both controller managers will 
+gain additional ~50 bytes because of the new lease under `resourceName`.
+
+###### Will enabling / using this feature result in increasing time taken by any operations covered by existing SLIs/SLOs?
+
+No.
+
+###### Will enabling / using this feature result in non-negligible increase of resource usage (CPU, RAM, disk, IO, ...) in any components?
+
+Both `kube-controller-manager` and `cloud-controller-manager` runs another leader election process,
+which may slightly increase CPU usages during leader changes, and it needs extra memory to hold another lease.
+
+### Troubleshooting
+
+###### How does this feature react if the API server and/or etcd is unavailable?
+
+The existing implementation of controller managers will `klog.Fatal` once the leader times out, which eventually happens if API server is unavailable.
+Leader Migration will not change such behavior.
+
+###### What are other known failure modes?
+
+None. Leader Migration is known to fail only because of misconfiguration or unavailability of the API server, both of which are discussed above.
+
+###### What steps should be taken if SLOs are not being met to determine the problem?
+N/A.
+
 ## Implementation History
 
 - 07-25-2019 `Summary` and `Motivation` sections were merged signaling SIG acceptance
 - 01-21-2019 Implementation details are proposed to move KEP to `implementable` state.
 - 09-30-2020 `LeaderMigrationConfiguration` and `ControllerLeaderConfiguration` schemas merged as #94205.
 - 11-04-2020 Registration of both types merged as #96133
 - 12-28-2020 Parsing and validation merged as #96226
+- 03-10-2021 Implementation for alpha state completed, released in 1.21.
+- 03-30-2021 User guide published as kubernetes/website#26970
+- 05-11-2021 KEP updated to target beta. 
```

Comment:
SIG Cloud Provider & the Cloud Provider Extraction WG agreed that having a default migration config for the standard case (HA migration KCM -> CCM with the 4 default controllers) was a Beta requirement. This doc should reflect that.

Reply:
Add the default migration configuration to the beta graduation criteria.